packages/cli-kit/assets/fetch-notifications.js

```diff
@@ -0,0 +1,25 @@
+import fetch from 'node-fetch'
+import Conf from 'conf'
```

Comment:
The bundle is not able to find these imports. Removing the need of `conf` would be hard, so I'll try to convert it to TS, compile and call it from the dist folder, so that it can use dependencies.